news/2021-03-18-community-contributors-2020.md

```diff
@@ -0,0 +1,246 @@
+
+![Community Contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg)
+
+Today, we welcome more into the fold, and perhaps some that should have been welcomed long ago.
+
+We present to you, the latest of the greatest:
+
+## Contributors
+
+[![](/wiki/shared/news/2021-03-18-community-contributors-2020/hallowatcher.jpg)](https://osu.ppy.sh/users/1874761)
+
+**hallowatcher** has had a long and storied history with osu!, but is most renowned for his creation of the now-ubiquitous 'pickems' games that accompany the official World Cups and a number of community tournaments as well. Pickems have since become something of a sensation, attracting well over 5000+ unique users and over 40 variants hosted, and several have even been the recipients of official World Cup profile badges, all while remaining a top tier tournament competitor himself.
+
+Not only that, **hallowatcher** has developed numerous enduring applications to enhance the lives of tournament organizers everywhere, from website frontends to assistance suites for the ever-taxed referees who help make the scene tick. He is also the creator of [chat4osu](https://osu.ppy.sh/community/forums/topics/879262), an application that retains a significant following to this day and remains one of the handful of apps that can survive the titan that 2018+ osu! has become.
+
+His efforts have quite literally, helped shape a scene for years to come and brought joy to tens of thousands of people. Belated though it may be, we finally welcome him aboard as a Community Contributor, and a most deserved one at that.
+
+[![](/wiki/shared/news/2021-03-18-community-contributors-2020/mangomizer.jpg)](https://osu.ppy.sh/users/1893718)
+
+When he isn't setting improbable feats of calculation and endurance as the self-titled "osu!taiko masochist", **mangomizer** is an ever-present aid in the World Cup scene, either hosting in or assisting with an absolute glut of projects intended to help brighten up people's day.
+
+A five year plus referee in the osu!taiko World Cup and the cryptic mastermind of many a World Cup bingo or the very aptly named osu! Trivium Quiz, **mangomizer** has without a doubt left a profoundly positive impact upon the community with his efforts, and stands starkly as a rare example of someone who seeks to build with and uplift others, embodying the core osu! spirit of "making things more awesome for everyone".
+
+[![](/wiki/shared/news/2021-03-18-community-contributors-2020/lasse.jpg)](https://osu.ppy.sh/users/896613)
+
+Though recently retired from the role, during his heyday **Lasse** remained consistently one of the most active members of the Nomination Assessment Team who still attended to the general duties of a normal Beatmap Nominator, advancing to stand at the very pinnacle of the overall kudosu ranking and consistently scoring a top 5 place among the BN in the process. 
```

Comment:
> ...scoring a top 5 place among the BN in the process. 

top 5 on what leaderboard? there are no bn rankings?

Reply:
it references the elite candidacy thing, but i'll juggle that around